<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LarkyCharmGui</class>
 <widget class="QMainWindow" name="LarkyCharmGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>244</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>177</red>
        <green>177</green>
        <blue>177</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>177</red>
        <green>177</green>
        <blue>177</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>177</red>
        <green>177</green>
        <blue>177</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>177</red>
        <green>177</green>
        <blue>177</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>LarkyCharm</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="lcd1">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>93</red>
              <green>93</green>
              <blue>93</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(93, 93, 93);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(93, 93, 93);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(93, 93, 93);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(93, 93, 93);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(93, 93, 93);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcd6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(93, 93, 93);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="lottoGoButton">
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="lottoClearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLabel" name="lottoLabel_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y E B O   M I L L I O N A R E S</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLineEdit" name="yeboEdit">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="yeboGoButton">
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="yeboClearButton">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lottoLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>L O T T O</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
